libs/staking/src/lib/components/validator-list.tsx

```diff
@@ -153,7 +153,7 @@ export function ValidatorList({
                   onChange={setInactiveValidatorsVisible}
                   value={isInactiveValidatorsVisible}
                 >
-                  {t('show-inactive', 'Show inactive')}
+                  {t('show-inactive', 'Show Inactive')}
```

Comment:
revert please

Reply:
why?

I was looking through the whole i18n PR and noticed that in original file it was capitalised https://github.com/haqq-network/frontend/pull/970/files#r1829697264